go/updater/updater.go

```diff
@@ -241,6 +241,7 @@ func (u *Updater) downloadAsset(asset keybase1.Asset) (fpath string, cached bool
 		}
 	}
 	u.log.Info("Moving %s to %s", filepath.Base(savePath), filepath.Base(fpath))
+	file.Close() // Couldn't rename otherwise on Windows
```

Comment:
Instead of doing this I might wrap the file save in a function and use defer for Close (cleanup). You could even use an anonymous function to do "inline".